code/datums/diseases/pierrot_throat.dm

```diff
@@ -26,3 +26,39 @@
 		if(4)
 			if(prob(5))
 				affected_mob.say( pick( list("HONK!", "Honk!", "Honk.", "Honk?", "Honk!!", "Honk?!", "Honk...") ) )
+
+
+/datum/disease/pierrot_throat/advanced
+	name = "Advanced Pierrot's Throat"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Banana products, especially banana bread."
+	cures = list("banana")
+	cure_chance = 75
+	agent = "H0NI<42.B4n4 Virus"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will probably drive others to insanity and go insane themselves."
+	severity = DANGEROUS
+
+/datum/disease/pierrot_throat/advanced/stage_act()
+	..()
+	switch(stage)
+		if(1)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You feel a very silly.</span>")
```

Comment:
Either `You feel a little silly` or `You feel very silly`

Reply:
you become the silliness

Reply:
You feel a little very silly

Reply:
little very kinda silly. Its a clown virus....